kube-derive/src/custom_resource.rs

```diff
@@ -152,18 +191,21 @@ pub(crate) fn derive(input: proc_macro2::TokenStream) -> proc_macro2::TokenStrea
         }
     }
 
-    // Schema generation is always enabled for v1 because it's mandatory.
-    // TODO Enable schema generation for v1beta1 if the spec derives `JsonSchema`.
-    let schema_gen_enabled = apiextensions == "v1" && cfg!(feature = "schema");
+    // Enable schema generation by default for v1 because it's mandatory.
+    let schema_mode = schema_mode.unwrap_or(if apiextensions == "v1" {
+        SchemaMode::Derived
+    } else {
+        SchemaMode::Disabled
+    });
```

Comment:
Disabled is still useful with v1, because schema is useless if you're not actually installing the `::crd()` (either because the CRD is "owned" by another application, or because you're maintaining the CRD YAML by hand).